modules/freeWheelAdserverVideo.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * This module adds Freewheel support for Video to Prebid.
+ */
+
+import { registerVideoSupport } from '../src/adServerManager';
+import { auctionManager } from '../src/auctionManager';
+import { groupBy, deepAccess } from '../src/utils';
+import { config } from '../src/config';
+// import { ADPOD, initAdpodHooks } from './adpod';
+
+const ADPOD = 'adpod'; // remove later when adpod module is merged; use above commented import instead
+const adPodTargetingKey = 'hb_price_industry_duration';
```

Comment:
Need to update the key based on recent conversation; should be `hb_pb_cat_dur`.

Should we setup a shared constant somewhere to ensure the key is read consistently across different files?  For example, I use this key in the `adpod` file.

Reply:
I updated the code in the `adpod` module; I'm exporting this var now: `TARGETING_KEY_PB_CAT_DUR`